angular2-jwt.ts

```diff
@@ -70,31 +74,37 @@ export class AuthHttp {
 
   request(method:RequestMethod, url:string, body?:string) {
 
+    var allHeaders = new Headers();
+
+    this._config.customHeaders.forEach((values, name, headers) => {
+      var value = this._config.customHeaders.get(name);
+      allHeaders.append(name, value);
+    });
+
     var options = new RequestOptions({
```

Comment:
Instead of instantiating a `new RequestOptions` and ignoring any `RequestOptions` the developer may have already set. Dep Inject the existing RequestOptions and merge it like Http does
https://github.com/angular/angular/blob/2.0.0-beta.0/modules/angular2/src/http/http.ts#L94

Then you can set RequestOptions like here, that will apply to both Http and AuthHttp
https://stackoverflow.com/questions/34502398/how-do-you-set-global-custom-headers-in-angular2